File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    /*gets the name of the user*/
    int n = 0;
    do
    {
        n = get_int("Height: ");
    } 
    while (n < 1 || n > 8);

    for (int i = 0; i < n; i++)
    {
        //for every line, spaces = height - current line - 1
        for (int j = n - i - 1; j > 0; j--)
        {
            printf(" ");
        }
        //hashes are just printed after spaces
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

s = " "
a = "#"

while True:
    ans = get_int("Heigth: ")
    if (ans <= 0 or ans > 8):
        continue
    else:
        for i in range(ans):
            print(f"{s*(ans-i-1)}{a*(i+1)}  {a*(i+1)}")
        break